### PR TITLE
Issue/239/create edit event view

### PR DESCRIPTION
**Pull request by Bedpaw**

### CODE REVIEW

File: src/utils/macros/routes.js [hunk 1]
```diff
@@ -79,4 +79,11 @@ export const ROUTES = {
     shortLabel: 'Mapa',
     icon: ICONS.map,
   },
+  editEvent: {
+    path: '/edit-event',
+    name: 'edit-event',
+    label: 'Edytowanie wydarzenia',
+    shortLabel: 'Wydarzenie',
+    icon: ICONS.map,
```
Comment (dbetka) [inline src/utils/macros/routes.js:87]: ICONS.edit

File: src/components/organisms/admin/edit-event.vue [hunk 2]
```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="f-flex f-flex-col f-flex-just-center">
+    <o-form :on-submit="updateEvent">
+      <m-input
+          :disabled="blockForm"
+          placeholder="Nazwa wydarzenia"
+          v-model="values.name"
+      />
+      <m-input
+          :disabled=true
```
Comment (dbetka) [inline src/components/organisms/admin/edit-event.vue]: `disabled` instead of `:disabled=true`

File: src/style/variables/colors/dark.sass [hunk 3]
```diff
@@ -5,6 +5,7 @@ $color-background-secondary: rgba(palette.$Purple, 0.3)
 $color-text: palette.$White
 $color-text-contrast: palette.$White
 $color-text-selected: palette.$White
+$color-text-disabled: rgba(palette.$Silver_Chalice, 0.5)
```
Comment (dbetka) [inline src/style/variables/colors/dark.sass:8]: Check in dark mode

File: src/structures/app-event.js [hunk 4]
```diff
@@ -7,7 +7,7 @@ export class AppEvent {
     mapZoom = 2,
   }) {
     this.eventId = eventId;
-    this.name = eventName;
+    this.eventName = eventName;
     this.mapPosition = {
```
Comment (dbetka) [inline src/structures/app-event.js:11]: It should like on backend

**Review by dbetka — CHANGES_REQUESTED** [review]
To fix

File: src/components/organisms/admin/edit-event.vue (hunk 2 above)
Comment (Bedpaw) [reply src/components/organisms/admin/edit-event.vue]: done

File: src/style/variables/colors/dark.sass (hunk 3 above)
Comment (Bedpaw) [reply src/style/variables/colors/dark.sass:8]: looks ok

File: src/utils/macros/routes.js (hunk 1 above)
Comment (Bedpaw) [reply src/utils/macros/routes.js:87]: done

File: src/structures/app-event.js (hunk 4 above)
Comment (Bedpaw) [reply src/structures/app-event.js]: changed

File: src/components/organisms/admin/edit-event.vue [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+      <m-input
+        :disabled="blockForm"
+        placeholder="Nazwa wydarzenia"
+        v-model="values.eventName"
+      />
+      <m-input
+        disabled
+        placeholder="Klucz wydarzenia"
+        v-model="values.eventId"
+      />
```
Comment (dbetka) [inline src/components/organisms/admin/edit-event.vue:13]: Add assist message for "Klucz wydarzenia" field (see Figma)

File: src/components/organisms/admin/edit-event.vue [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    updateEvent () {
+      const updatedEvent = {
+        ...this.getEventBasicInformation,
+        eventName: this.values.eventName,
+      };
+      api.updateEvent(updatedEvent)
+        .then(api.getEventById)
+        .then(this.setEvent)
+        .catch(this.onErrorOccurs);
+    },
```
Comment (dbetka) [inline src/components/organisms/admin/edit-event.vue:73]: Change this.message to 'Zapisanie nowych danych wydarzenia powiodło się.' and make focus out for save button on success.

File: src/store/event.js [hunk 7]
```diff
@@ -65,15 +61,24 @@ export default {
         return expirationTimeDiffNow > 0 && expirationTimeDiffNow < timeRange;
       });
     },
+    getEventBasicInformation: (state) => ({
```
Comment (dbetka) [inline src/store/event.js:64]: `getEventBasicInformation` -> `getEventBasicInformations`

File: src/components/organisms/admin/edit-event.vue [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+  },
+  methods: {
+    updateEvent () {
+      const updatedEvent = {
+        ...this.getEventBasicInformation,
+        eventName: this.values.eventName,
+      };
+      api.updateEvent(updatedEvent)
+        .then(api.getEventById)
+        .then(this.setEvent)
```
Comment (dbetka) [inline src/components/organisms/admin/edit-event.vue:71]: this.setEvent doesn't work properly:

Example step-by-step:
1. sign in on admin account
2. go to /edit-event
3. put new event name
4. click save button
5. click on start button in footer
6. there is old event name

File: src/utils/macros/errors.js [hunk 9]
```diff
@@ -11,4 +11,5 @@ export const ERRORS = {
     'Skontaktuj się z administratorem wydarzenia.',
   signOut: 'Sign out went wrong',
   collectPoint: 'Collecting point went wrong',
+  updateEvent: 'Something went wrong, event has not been updated',
```
Comment (dbetka) [inline src/utils/macros/errors.js:14]: Polish message put here

**Review by dbetka — CHANGES_REQUESTED** [review]

File: src/store/event.js (hunk 7 above)
Comment (Bedpaw) [reply src/store/event.js:64]: sorry, "information" is uncountable 😄

File: src/utils/macros/errors.js (hunk 9 above)
Comment (Bedpaw) [reply src/utils/macros/errors.js:14]: done

File: src/components/organisms/admin/edit-event.vue (hunk 5 above)
Comment (Bedpaw) [reply src/components/organisms/admin/edit-event.vue:13]: done

File: src/store/event.js (hunk 7 above)
Comment (dbetka) [reply src/store/event.js:64]: Right xD

**Review by dbetka — APPROVED** [review]

**Pull request merged**